index.d.ts

```diff
@@ -128,8 +132,8 @@ export interface DailyParticipant {
   owner: boolean;
 
   // video element info (iframe-based calls using standard UI only)
-  cam_info: DailyVideoElementInfo;
-  screen_info: DailyVideoElementInfo;
+  cam_info: {} | DailyVideoElementInfo;
+  screen_info: {} | DailyVideoElementInfo;
```

Comment:
The intent here actually is to signal an "all-or-nothing" field. Either you'll get `{}` or an object with all of the (non-optional) properties in `DailyVideoElementInfo`. It's slightly unwieldy to work with, I know (it means you have to do something like a runtime check for the property you're interested in and then cast to `DailyVideoElementInfo` to access it), but it's more accurate.